src/license/src/lib.rs

```diff
@@ -20,3 +20,26 @@ mod manager;
 pub use feature::*;
 pub use key::*;
 pub use manager::*;
+use risingwave_pb::telemetry::PbTelemetryEventStage;
+use risingwave_telemetry_event::report_event_common;
+
+pub(crate) fn report_telemetry(feature_name: &str, success_flag: bool) {
+    if !feature_name.eq_ignore_ascii_case("TestPaid") {
```

Comment:
Better to use `enum` (and `==`) instead of string comparing. 

This is more of a code style issue, because it's Rust not JS.